### PR TITLE
feat: add rating controller

### PR DESCRIPTION
**Pull request by 0xff-dev**
#### What type of PR is this?
feat: add rating controller

#### Which issue(s) this PR fixes

Fix: https://github.com/kubebb/core/issues/249

### CODE REVIEW

File: charts/kubebb-core/templates/pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,136 @@
+{{- if .Values.deployment.rating_enable }}
+apiVersion: tekton.dev/v1beta1
```
Comment (bjwswang) [inline charts/kubebb-core/templates/pipeline.yaml:2]: @0xff-dev Can we also check if tekton crds exists? 

Like `{{ if .Capabilities.APIVersions.Has "tekton.dev/v1beta1" -}}`

**Review by bjwswang — DISMISSED** [review]

**Review by bjwswang — COMMENTED** [review]
We need to update the default image tag in values.yaml

**Comment by bjwswang** [comment]
> 

I can release a new version `v0.1.4`. And you take this as the default image tag.

**Review by Abirdcfly — APPROVED** [review]

**Comment by bjwswang** [comment]
@0xff-dev update the image to v0.1.4

File: charts/kubebb-core/templates/pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        # Download and untar chart package
+        helm pull --untar $(params.url)
+
+        helm lint ./$(params.component)
+        echo $? | tee $(results.LINT.path)
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name:  {{ .Release.Name }}
```
Comment (bjwswang) [inline charts/kubebb-core/templates/pipeline.yaml:72]: We have two pipelines now 
- [component-reliability](https://github.com/kubebb/core/blob/main/pipeline/rating/reliability/pipeline.yaml)
- [component-security](https://github.com/kubebb/core/blob/main/pipeline/rating/security/pipeline.yaml)

**Review by Abirdcfly — APPROVED** [review]

**Review by Abirdcfly — DISMISSED** [review]

File: charts/kubebb-core/templates/pipeline-reliability.yaml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        # Download and untar chart package
+        helm pull --untar $(params.url)
+
+        helm lint ./$(params.component)
+        echo $? | tee $(results.LINT.path)
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name:  {{ .Release.Name }}
```
Comment (bjwswang) [inline charts/kubebb-core/templates/pipeline-reliability.yaml:32]: `{{ .Relase.Name }}-reliability`

File: charts/kubebb-core/templates/pipeline-security.yaml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+        cat <<EOF |jq --arg dot "$(cat r.dot |base64)" '.binaryData.r = $dot' | jq --arg name "$NAME" '.metadata.name = $name' | jq --arg namespace "$NAMESPACE" '.metadata.namespace = $namespace' | kubectl apply -f -
+        {"kind": "ConfigMap","apiVersion": "v1","metadata": {"name": "configmap","namespace": "default"},"binaryData": {"r": ""}}
+        EOF
+        echo ${NAME} | tee $(results.RBACCM.path)
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name:  {{ .Release.Name }}
```
Comment (bjwswang) [inline charts/kubebb-core/templates/pipeline-security.yaml:45]: `{{ .Release.Name }}-security`

**Review by Abirdcfly — APPROVED** [review]

**Review by bjwswang — APPROVED** [review]

**Pull request merged**